install/install.sh

```diff
@@ -60,33 +82,32 @@ echo "-------------------------------"
 echo ""
 echo "In order to configure the platform, we need some information from you. Please reply to the following questions:"
 echo ""
-read -s -e -r -p  "Password of the 'root' user of MySQL: " -i "" db_root_password
+read -s -e -r -p  "$(tput bold)Password of the 'root' user of MySQL:$(tput sgr0) " -i "" db_root_password
 echo ""
 # Verify password
 
 supress_warning=$(mysql_config_editor set --login-path=root_login --host=localhost --user=root --password "${db_root_password}") >> "$install_log" 2>&1
```

Comment:
Currently, this doesn't work, I am forced to use a blank password, it says password authentication failed.
I'll try to fix this in this PR only...

Reply:
@Tarun-Arora Let me know when you have fixed the error.

Reply:
@thealphadollar I have fixed the issue.